tests/networking/helper/helper.go

```diff
@@ -89,6 +97,21 @@ func DefineAndCreateDeamonsetWithMultusAndSkipLabelOnCluster(nadName string) err
 	return defineDaemonSetBasedOnArgs(nadName, tsparams.NetworkingTestMultusSkipLabel)
 }
 
+// DefineAndCreateDeploymentOnCluster defines deployment resource and creates it on cluster.
+func DefineAndCreateDeploymentWithContainerPorts(replicaNumber int32, ports []corev1.ContainerPort) error {
+	deploymentUnderTest, err := defineDeploymentWithContainers(replicaNumber, len(ports), tsparams.TestDeploymentAName)
+
```

Comment:
I'm surprised the linter didn't flag this extra whitespace 😄

Reply:
lol this linter is usually concerned about not having enough blank lines between things